include/libtorrent/config.hpp

```diff
@@ -431,23 +431,23 @@ int snprintf(char* buf, int len, char const* fmt, ...)
 #ifdef _GLIBCXX_USE_NOEXCEPT
 #define TORRENT_EXCEPTION_THROW_SPECIFIER _GLIBCXX_USE_NOEXCEPT
 #else
-#if __cplusplus <= 199711L || defined BOOST_NO_CXX11_NOEXCEPT
-#define TORRENT_EXCEPTION_THROW_SPECIFIER throw()
-#else
+#if !defined BOOST_NO_CXX11_NOEXCEPT && (__cplusplus > 199711L || (defined _MSC_VER && _MSC_VER >= 1900)) 
```

Comment:
I intend to require c++11 in master (travis and appveoyr already dropped c++98 builds)